packages/core/src/card/InteractableCardGroup.tsx

```diff
@@ -0,0 +1,206 @@
+import {
+  ComponentPropsWithoutRef,
+  forwardRef,
+  KeyboardEvent,
+  SyntheticEvent,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from "react";
+import { clsx } from "clsx";
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+
+import { makePrefixer, useControlled, useForkRef } from "../utils";
+import {
+  InteractableCardGroupContext,
+  SelectionVariant,
+  InteractableCardValue,
+} from "./InteractableCardGroupContext";
+import interactableCardGroupCss from "./InteractableCardGroup.css";
+
+export interface InteractableCardGroupProps
+  extends Omit<ComponentPropsWithoutRef<"div">, "onChange"> {
+  /**
+   * The default value. Use when the component is not controlled. Should be an array when `selectionVariant` is "multiselect".
+   */
+  defaultValue?: InteractableCardValue;
+  /**
+   * If `true`, the Interactable Card Group will be disabled.
+   */
+  disabled?: boolean;
+  /**
+   * The value. Use when the component is controlled.
+   */
+  value?: InteractableCardValue;
+  /**
+   * The selection variant.
+   */
+  selectionVariant?: SelectionVariant;
```

Comment:
I would go for 'multiselect' prop as boolean rather than selectionVariant, seems more intuitive esp around what we use 'variant' for elsewhere

Reply:
I was deciding between the two, calendar has a `selectionVariant` prop but there are more than 2 so it makes more sense.